model_analyzer/config/generate/model_run_config_generator.py

```diff
@@ -57,49 +57,39 @@ def __init__(self, config, gpus, model, client, variant_name_manager,
 
         self._determine_early_exit_enables(config, model)
 
-        # This prevents an error when is_done() is checked befored the first next_config() call
-        self._pacg = PerfAnalyzerConfigGenerator(self._config, self._model_name,
-                                                 self._model_pa_flags,
-                                                 self._model_parameters,
-                                                 self._pacg_early_exit_enable)
-
         self._mcg = ConfigGeneratorFactory.create_model_config_generator(
             self._config, self._gpus, model, self._client,
             self._variant_name_manger, default_only,
             self._mcg_early_exit_enable)
 
         self._curr_mc_measurements = []
 
-    def is_done(self):
-        return self._pacg.is_done() and self._mcg.is_done()
+    def _is_done(self):
```

Comment:
We shouldn't be reaching into member variables to use their private methods. Do we even need this anymore with the new code format?  

I'm realizing now that the _is_done() probably shouldn't be part of the interface, as no consumer should ever use it.